ska_sun/sun.py

```diff
@@ -72,12 +88,19 @@ def allowed_rolldev(pitch):
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-def position(time):
+def position_fast(time):
     """
-    Calculate the sun position at the given ``time``.
+    Calculate the sun position at the given ``time`` using a fast approximation.
 
     Code modified from http://idlastro.gsfc.nasa.gov/ftp/pro/astro/sunpos.pro
 
+    This implementation is returns coordinates that are in error by as much as
+    0.3 deg. Use the ``position()`` function unless speed is critical.
```

Comment:
The `position` function is now basically a wrapper, so should the help in `position_fast` say to use `position_accurate` or `position` with config configured to be accurate?

Reply:
Good catch, this should recommend using `position(..., method="accurate")`. Thinking more about potential confusion, it might be best to make these two functions be private and not exported in the API.